src/components/MainContainer.tsx

```diff
@@ -70,14 +71,22 @@ const MainContainer = () => {
     return instructions;
   };
 
+  const addIngredient = (ingredient: string) => {
```

Comment:
should this function have a return type of void since it is not returning anything?